keyboards/handwired/dactyl_manuform/4x6/keymaps/scheikled/rules.mk

```diff
@@ -1,10 +0,0 @@
-# Copyright 2021 Paul Maria Scheikl (@ScheiklP)
```

Comment:
Why are you deleting this user keymap?

Reply:
I'm not versed enough in either Git nor QMK to understand how to get the build checks to pass if this keymap wasn't removed.
In `keymap.c` of said user keymap, `#include "koy_keys_on_quertz_de_latin1.h"` and keycodes referencing it were causing build checks to fail.

Reply:
That doesnt mean you should just remove the keymap....

Reply:
If I were to restore said user keymap, and build check fails, is it on me to fix something I've had no involvement in? i.e. is all passes on build checks required for PR to be merged?

Reply:
deleted keymap restored